docs/requirements.txt

```diff
@@ -0,0 +1 @@
+sphinx-astropy
```

Comment:
I feel it would be more safe to list all dependencies here, but as there are no optional dependencies this should be fine to be picked up from setup.py

Reply:
Yeah I was tempted to add more, like sphinx and other things, but also sphinx-astropy is hopefully good enough to know what it needs.  We can add in anything else as we need to